main.js

```diff
@@ -1,55 +1,65 @@
-//----------------5.0.----------------
-const obj1 = {
-    a: 'a',
-    b: {
-        a: 'a',
-        b: 'b',
-        c: {
-            a: 1,
-        },
+//_________________5.0_________________
+const studentsData = [
+    {
+        firstName: 'Василий',
+        lastName: 'Петров',
+        admissionYear: 2019,
+        courseName: 'Java',
     },
-};
-const obj2 = {
-    b: {
-        c: {
-            a: 1,
-        },
-        b: 'b',
-        a: 'a',
+    {
+        firstName: 'Иван',
+        lastName: 'Иванов',
+        admissionYear: 2018,
+        courseName: 'JavaScript',
     },
-    a: 'a',
-};
-const obj3 = {
-    a: {
-        c: {
-            a: 'a',
-        },
-        b: 'b',
-        a: 'a',
+    {
+        firstName: 'Александр',
+        lastName: 'Федоров',
+        admissionYear: 2017,
+        courseName: 'Python',
     },
-    b: 'b',
-};
+    {
+        firstName: 'Николай',
+        lastName: 'Петров',
+        admissionYear: 2019,
+        courseName: 'Android',
+    }
+];
+
+class User {
+    constructor(firstName, lastName) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+    get fullName() {
+        return `${this.firstName} ${this.lastName}`
+    }
+}
+
+class Student extends User {
+    constructor(firstName, lastName, admissionYear, courseName) {
+        super(firstName, lastName);
+        this.admissionYear = admissionYear;
+        this.courseName = courseName;
+    }
+    get course() {
+        return 2021 - this.admissionYear;
```

Comment:
Lets change `2021` into some value from `Date` object. For example: `const currentYear = new Date().getFullYear();`

Reply:
Ok, thank you !